modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-impl/src/main/java/com/liferay/headless/commerce/delivery/catalog/internal/resource/v1_0/ProductResourceImpl.java

```diff
@@ -124,6 +126,41 @@ public Product getChannelProduct(
 		return _toProduct(commerceContext, cpDefinition);
 	}
 
+	@Override
+	public Product getChannelProductByFriendlyUrlPath(
+			Long channelId, String friendlyUrl, Long accountId)
```

Comment:
in the yaml file it was called `friendlyUrlPath` here you called it `friendlyUrl`